src/audio/igo_nr/igo_nr.c

```diff
@@ -360,48 +497,42 @@ static int igo_nr_free(struct processing_module *mod)
 	return 0;
 }
 
-#if DISABLED_CODE
```

Comment:
ok, I see now, for the final version would be good to have it a bit better to avoid breaking bisection

Reply:
Yep, I'll work out the intermediate version from this final. The function  wasn't essential for running the component so I just disabled it for first version.